LiteCore/Storage/UnicodeCollator.cc

```diff
@@ -50,6 +58,59 @@ namespace litecore {
         });
     }
 
+    int LikeUTF8(slice comparand, slice pattern, const Collation& col) {
```

Comment:
It's a good idea to add a comment saying where this code came from (inside SQLite, right?)